src/sidebar/components/hypothesis-app.js

```diff
@@ -150,30 +149,32 @@ function HypothesisAppController(
   const promptToLogout = function() {
     // TODO - Replace this with a UI which doesn't look terrible.
     let text = '';
-    if (drafts.count() === 1) {
+    const drafts = store.countDrafts();
+    if (drafts === 1) {
       text =
         'You have an unsaved annotation.\n' +
         'Do you really want to discard this draft?';
-    } else if (drafts.count() > 1) {
+    } else if (drafts > 1) {
       text =
         'You have ' +
-        drafts.count() +
+        drafts +
         ' unsaved annotations.\n' +
         'Do you really want to discard these drafts?';
     }
-    return drafts.count() === 0 || $window.confirm(text);
+    return drafts === 0 || $window.confirm(text);
   };
 
   // Log the user out.
   this.logout = function() {
     if (!promptToLogout()) {
       return;
     }
+
     store.clearGroups();
-    drafts.unsaved().forEach(function(draft) {
+    store.unsavedAnnotations().forEach(function(draft) {
```

Comment:
The callback parameter name (`draft`) was incorrect / misleading before as `draft` here is actually an annotation with no properties other than `$tag` and `id`. I'd suggest renaming to `annotation` or destructuring to `{ id, $tag }`.

Reply:
good catch